lib/scrabble.rb

```diff
@@ -1,7 +1,13 @@
 class Scrabble
 
   def score(word)
-    1
+    return 0 if word == nil
+    word_upcase = word.upcase.chars
+    sum = 0
+      word_upcase.each do |letter|
```

Comment:
Indentation here is off. Indent the block (everything within the `do...end`), but not the lines surrounding it.